docs/source/_static/juju/bundle.yaml

```diff
@@ -0,0 +1,155 @@
+envExport:
+  services:
+    cinder:
+      charm: "cs:trusty/cinder-136"
+      num_units: 1
+      annotations:
+        "gui-x": "1045.973572253264"
+        "gui-y": "544.8983810601128"
+      to:
+        - "calico-acl-manager=0"
+    "neutron-calico":
+      charm: "cs:~cory-benfield/trusty/neutron-calico-7"
```

Comment:
Are all the charms still in your private space?  Can/should we create an official Calico location as we did with the PPAs?